src/main/groovy/de/dkfz/roddy/execution/cluster/pbs/PBSJobManager.groovy

```diff
@@ -89,7 +89,7 @@ class PBSJobManager extends ClusterJobManager<PBSCommand> {
     }
 
     PBSCommand createCommand(BEJob job) {
-        return new PBSCommand(this, job, job.jobName, [], job.parameters, [:], [], job.dependencyIDsAsString, job.tool.getAbsolutePath(), job.getLoggingDirectory())
+        return new PBSCommand(this, job, job.jobName, [], job.parameters, [:], [], job.dependencyIDsAsString, job.tool?.getAbsolutePath() ?: job.getToolScript(), job.getLoggingDirectory())
```

Comment:
Here you priorized the tool over the pipe. However, the string is never used in the PBSCommand toString() method. It's ok for now.